lib/drivers/medtronic600Driver.js

```diff
@@ -0,0 +1,1418 @@
+  constructor(pumpSession) {
+    super(MinimedMessage.COMMAND_TYPE.OPEN_CONNECTION, pumpSession, pumpSession.getHMAC(),
+      OpenConnectionResponse);
+  }
+}
+
+class CloseConnectionResponse extends MinimedResponse {}
+
+class CloseConnectionRequest extends MinimedRequest {
+  constructor(pumpSession) {
+    super(MinimedMessage.COMMAND_TYPE.CLOSE_CONNECTION, pumpSession, pumpSession.getHMAC(),
+      CloseConnectionResponse);
+  }
+}
+
+class ReadInfoResponse extends MinimedResponse {
+  // Link MAC and Pump MAC are packed as a 64-bit integers, but JavaScript doesn't support those,
+  // so we'll store them as a hex strings.
+  get linkMAC() {
+    const offset = MinimedMessage.ENVELOPE_SIZE + 0x00;
+    return this.payload.slice(offset, offset + 8).toString('hex');
+  }
+
+  get pumpMAC() {
+    const offset = MinimedMessage.ENVELOPE_SIZE + 0x08;
+    return this.payload.slice(offset, offset + 8).toString('hex');
+  }
+
+  get linkCounter() {
+    return this.payload.readUInt16LE(MinimedMessage.ENVELOPE_SIZE + 0x10);
+  }
+
+  get encryptionMode() {
+    // eslint-disable-next-line no-bitwise
+    return this.payload[MinimedMessage.ENVELOPE_SIZE + 0x12] & 1;
+  }
+
+  get isAssociated() {
+    return this.pumpMAC !== '0000000000000000';
+  }
+}
+
+class ReadInfoRequest extends MinimedRequest {
+  constructor(pumpSession) {
+    super(MinimedMessage.COMMAND_TYPE.READ_INFO, pumpSession, null, ReadInfoResponse);
+  }
+}
+
+class RequestLinkKeyResponse extends MinimedResponse {
+  get packedLinkKey() {
+    return this.payload.slice(MinimedMessage.ENVELOPE_SIZE, MinimedMessage.ENVELOPE_SIZE + 55);
+  }
+
+  linkKey(cnlModelAndSerial) {
+    const key = Buffer.alloc(16);
+
+    // eslint-disable-next-line no-bitwise
+    let pos = cnlModelAndSerial.slice(-1) & 7;
+
+    /* eslint-disable no-bitwise */
+    for (let i = 0; i < key.length; i++) {
+      if ((this.packedLinkKey[pos + 1] & 1) === 1) {
+        key[i] = ~this.packedLinkKey[pos];
+      } else {
+        key[i] = this.packedLinkKey[pos];
+      }
+
+      if (((this.packedLinkKey[pos + 1] >> 1) & 1) === 0) {
+        pos += 3;
+      } else {
+        pos += 2;
+      }
+    }
+    /* eslint-enable no-bitwise */
+
+    return key;
+  }
+}
+
+class RequestLinkKeyRequest extends MinimedRequest {
+  constructor(pumpSession) {
+    super(MinimedMessage.COMMAND_TYPE.REQUEST_LINK_KEY, pumpSession, null, RequestLinkKeyResponse);
+  }
+}
+
+class JoinNetworkResponse extends TelDResponse {
+  get radioChannel() {
+    if (this.payload.length > 46 && this.payload[0x33] === 0x82 && this.payload[0x44] === 0x42) {
+      return this.payload[0x4c];
+    }
+
+    return 0;
+  }
+
+  get joinedNetwork() {
+    return this.radioChannel !== 0;
+  }
+}
+
+class JoinNetworkRequest extends TelDRequest {
+  static get READ_TIMEOUT_MS() {
+    return 10000;
+  }
+
+  constructor(pumpSession) {
+    const payloadBuffer = Buffer.alloc(26);
+
+    /* eslint-disable lodash/prefer-lodash-method */
```

Comment:
Lodash is being used elsewhere in the file. This eslint rule is asking "Why are you using the object's `fill` method, when you could use `lodash`?". I'm using the object's fill method because it looks more consistent in this block of code.